tests/test_definitions/test_expectations_v2_api.py

```diff
@@ -287,13 +285,14 @@ def pytest_generate_tests(metafunc):
 @pytest.mark.order(index=1)
 @pytest.mark.integration
 @pytest.mark.slow  # 14.90s
-def test_case_runner(test_case):
+@pytest.mark.v2_api
```

Comment:
@anthonyburdi in line 158 of this file, you may as well rename bigquery_v2 to bigquery_v2_api to be consistent with how bigquery_cfe was renamed. The bigquery_v2 marker doesn't appear to be used in any test_definition JSON files, but there is code that checks for it in case a marker is added in the future.

You should add a conditional for checking bigquery_v2_api around line 244 as well